packages/cli/test/cases/build.data.graph/src/templates/blog-template.js

```diff
@@ -50,7 +50,7 @@ class BlogTemplate extends LitElement {
             ${posts.map((item) => {
               return html`
                 <li>
-                  <a href="${item.link}/" title="Click to read my ${item.title} blog post">${item.title}</a>
+                  <a href="${item.link}/" title="Click to read my ${item.title} page">${item.title}</a>
```

Comment:
as mentioned above, not sure if this actually needs to be changed?

Reply:
Ah, for some reason I thought title was "inferred" / generated by the graph lifecycle, but that is [not the case](https://github.com/ProjectEvergreen/greenwood/blob/release/0.5.0/packages/cli/src/lifecycles/graph.js#L75) as I had thought.  

Maybe it could / should be?  (can make a new issue for that)

Reply:
reverted back to original